hermes/probe/probe.go

```diff
@@ -149,22 +140,24 @@ func (mp *Probe) runProbe(ctx context.Context, metricChan chan<- *cpmetrics.Even
 	var wg sync.WaitGroup
 	for _, t := range mp.targets {
 		wg.Add(1)
-		go func(t *Target) {
+		go func(t *target.Target) {
```

Comment:
Not that this is wrong, but there's another pattern for this:
```
for _, t := range mp.targets {
  t := t // Yes, this looks surprising. It creates a new locally-scoped variable called `t`. Otherwise the loop variable `t` might be changed (iterated) when the goroutine actually executes, so it will use the wrong value.
  go func() {
    // Just use `t` here.
  }
}
```

Reply:
I prefer this pattern as it seems a bit more logical for me, so I've changed it accordingly.